src/main/webapp/repo-bootstrap/global/configuration/samples/sample-site-config.xml

```diff
@@ -21,11 +21,34 @@
     <display-name>{siteName}</display-name>
     <default-timezone>EST5EDT</default-timezone>
 
+    <form-engine>
+        <!-- Indicates if postfixes should be required for all fields -->
+        <field-name-postfix>true</field-name-postfix>
+        <!-- List of field names that should not have a postfix -->
+        <ignore-postfix-fields>
+            <field>internal-name</field>
+            <field>file-name</field>
+            <field>placeInNav</field>
+            <field>scripts</field>
+            <field>mime-type</field>
+            <field>force-https</field>
+            <field>navLabel</field>
+            <field>expired</field>
+            <field>key</field>
+            <field>value</field>
+            <field>items</field>
+            <field>redirect-url</field>
+            <field>authorizedRoles</field>
+            <field>role</field>
+            <field>disabled</field>
+        </ignore-postfix-fields>
+    </form-engine>
+
     <!--
-        Pattern that Studio will use to load extensions from the site repository
-        Required placeholders: ${module}, ${type}, ${name}
+        Pattern that Studio will use to load plugin from the site repository
+        Required placeholders: ${type}, ${name}
      -->
-    <extension-folder-pattern>/config/${module}/extension/${type}/${name}</extension-folder-pattern>
+    <plugin-folder-pattern>/config/studio/plugins/${type}/${name}</plugin-folder-pattern>
```

Comment:
Why this change?

Reply:
It's something I noticed was wrong, but it's unrelated